plugins/modules/cloudformation_info.py

```diff
@@ -91,27 +91,27 @@
 - set_fact:
     stack_name: my-awesome-stack
 
-- cloudformation_info:
```

Comment:
You've missed 'cloudformation_facts' on line 84 (should also be changed to cloudformation_info)

Reply:
@Akasurde Is this PR using https://github.com/ansible-network/collection_prep? We might need to fix update.py for info/facts (or anything symlinked) modules.